terraform/iam_module/main.tf

```diff
@@ -0,0 +1,47 @@
+resource "google_storage_bucket" "catalog_bucket" {
+  project  = var.project
+  location = var.location
+  name     = "catalog-${var.project}"
+  labels = {
+    team = var.team_name, purpose = "Catalog bucket for data products on DASK"
```

Comment:
For meg synes jeg det er lettere å lese når hvert key-value par er på hver sin linje, men det er opp til deg

Reply:
Godt poeng, kan formatere det bedre.